Editor/Analytics/AnalyticsEditorLogger.cs

```diff
@@ -16,16 +18,14 @@ public static void Enable()
         {
             IsEnabled = true;
```

Comment:
I know it's not in your PR, but you could simplify this by removing the local IsEnable variable manual setting, and then on line 15 just make it a computed property... If Unity supports c# computed properties 😅 

`public static bool IsEnabled => CoreSettingsHandler.CoreSettings.EnableAnalytics;`

Reply:
right good idea i guess it should work, just not sure if it will update or it only gets the value initially 🤔 
Will test it out

Reply:
actually one potential issue I see with this ^ 
Is that order of operations might cause problems when initially installing the package. 
As we create the CoreSettings scriptable object from C# after the package is imported. I assume that this variable would be computed before the SO (scriptable object) is created so it would return null when trying to load it.

Reply:
So it works if core settings already exists. 
But on first time install it tries to access the core settings (due to this code change) before core settings is created which causes an error. So I will keep the current logic with the extra variable.

Reply:
Ah, that sucks, but fair enough 👍